src/views/Package/components/OperatorArea/OperatorArea.tsx

```diff
@@ -74,6 +81,12 @@ export const OperatorArea: FunctionComponent<OperatorAreaProps> = ({
       items.push(<>Registry: {registryLink}</>);
     }
 
+    items.push(
```

Comment:
The location of the link feels out of place to me. It's different than other links we show here and I think deserves a place of its own. This also makes it hard to notice it. 

Can we move to the top right corner of the panel?

Reply:
I can move it to the top, but placing it top right seems out of place